src/js/api/api-mutators.js

```diff
@@ -70,6 +70,7 @@ define([
             'setFullscreen',
             'setVolume',
             'setMute',
+            'setPlaylist',
```

Comment:
Adding an API method? If so that should be clearer in the PR title, and the implications should be discussed - like how is this different than `load()`?

If it's not needed publicly, don't add it here.